### ISSUE TITLE
Git clone failed when we compile project use the Makefile.

### ISSUE THREAD

**Issue by EAHITechnology** [topling/toplingdb#9]
### Dependent information
Branch : All
Os : ubuntu 20.04

### Expected behavior
The git  pull repositories use `https://` schema not `ssh@` schema when I `make` the Makefile. 

### Actual behavior
In toplingdb Makefile, git use `ssh@` schema to pull repositories. On a machine without rsa we would get error:

```
Please make sure you have the correct access rights
and the repository exists.
+ cd sideplugin
+ git clone git@github.com:topling/cspp-memtable
Cloning into 'cspp-memtable'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
+ cd sideplugin
+ git clone git@github.com:topling/cspp-wbwi
Cloning into 'cspp-wbwi'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.
```

### Steps to reproduce the behavior
After resolving dependencies
```
cd toplingdb/
make -j 20
```

**Comment by EAHITechnology** [comment]
And `git@github.com:rockeet/topling-rocks` is not found, If the project does not depend on this repository, pls remove this download behavior.

**Comment by rockeet** [comment]
> And `git@github.com:rockeet/topling-rocks` is not found, If the project does not depend on this repository, pls remove this download behavior.

`topling-rocks` is an optional repo, fail to clone it does not affect ToplingDB integrity, we keep it in Makefile for internal use.

**Comment by rockeet** [comment]
Resolved by commit 0bf017f13e0a8db7b8934fe49a01a5ba75c414fd
